src/ParallelLaunch.cpp

```diff
@@ -145,6 +145,7 @@ static void * thread_main (void *lp)
 		auto now = chrono::high_resolution_clock::now();
 		// update interval in case it's a test with undertermined length
 		gtc->interval = chrono::duration_cast<chrono::microseconds>(now - gtc->start).count()/1000000.0;
+		std::cout<<"interval:"<<gtc->interval<<std::endl;
```

Comment:
Let's not put cout into the master branch

Reply:
I think it's somewhat important; I'll put it under `gtc->verbose` condition.